src/server/v2.0/handler/gc.go

```diff
@@ -28,6 +28,13 @@ func newGCAPI() *gcAPI {
 	}
 }
 
+func (g *gcAPI) Prepare(ctx context.Context, operation string, params interface{}) middleware.Responder {
+	if err := g.RequireSysAdmin(ctx); err != nil {
+		g.SendError(ctx, err)
```

Comment:
should be `return g.SendError(ctx, err)`

Reply:
both are fine, but updated.